### DISCUSSION TITLE
Problem with Stata Licence

### DISCUSSION THREAD

**Discussion by RaspberryEmma** [opensafely/documentation#1675] [Q&A]
Hi all,

I was working through the OpenSafely documentation example for generating a basic Stata model and encountered the following message:

![Screenshot from 2024-11-21 10-12-28](https://github.com/user-attachments/assets/610277c4-29be-447d-8000-1d7c296fb6ce)

The error message indicates that even though I'm an OpenSafely member, the corresponding Stata licence hasn't been able to verify.
What should I do?

(Additional context:  I'm a PhD student with the University of Bristol who I believe use a shared Stata licence.

**Comment by Jongmassey** [comment]
Hi Emma,

Can you confirm whether you're seeing this error when working in a codespace or working locally?

**Reply by RaspberryEmma** [reply]
Hiya,

Thank you for getting back to me, I'm seeing the error when running on GitHub codespaces, specifically via FireFox if that makes any difference.

**Reply by Jongmassey** [reply]
Hi Emma, is [this](https://github.com/opensafely/post-covid-cvd-methods) the repository you're working on, or a fork in your own user organisation? Did you see a dialogue for additional permissions as per [this page](https://docs.opensafely.org/getting-started/how-to/troubleshoot-common-codespaces-issues/#this-codespace-is-requesting-additional-permissions) in the docs? This permission is required to make the codespace work with the Stata license issued to the Bennett Institute, if you fork the repo into your own org and start the codespace from there, this permission won't work. 

If it is the above linked repo, it looks like this is a clone of an old repo that doesn't have the required codespaces configuration (part of which is what makes the stata licensing work) that is in the [research repository template](https://github.com/opensafely/research-template). The instructions to add the codespaces configuration to the repo are in [here](https://docs.opensafely.org/getting-started/how-to/add-github-codespaces-to-your-project/). If you add this configuration, you'll need to start a new codespace for it to work.
